build.xml

```diff
@@ -1,126 +1,417 @@
+        env.TRAVIS:       ${env.TRAVIS}
+    </echo>
+    <travis target="env" />
 
     <property name="src.dir"     value="src"/>
-    <property name="test.dir"    value="tests"/>
-    <property name="lib.dir"     value="libs" />
+    <property name="src_gen.dir" value="src_gen"/>
+    <property name="lib.dir"     location="libs" />
+    <property name="unit.dir"    value="tests/som" />
+    <property name="corelib.dir" location="core-lib" />
+    <property name="graal.dir"   location="${lib.dir}/truffle/compiler" />
+    <property name="truffle.dir" location="${lib.dir}/truffle/truffle" />
+    <property name="sdk.dir"     location="${lib.dir}/truffle/sdk" />
+    <property name="tools.dir"   location="${lib.dir}/truffle/tools" />
+    <property name="svm.dir"     location="${lib.dir}/truffle/substratevm" />
+    <property name="compiler.build" location="${graal.dir}/mxbuild/dists/jdk1.8" />
+    <property name="sdk.build"   location="${sdk.dir}/mxbuild/dists/jdk1.8" />
+    <property name="tools.build" location="${tools.dir}/mxbuild/dists/jdk1.8" />
+    <property name="svm.build"   location="${svm.dir}/mxbuild/dists/jdk1.8" />
+    <property name="truffle.build" location="${truffle.dir}/mxbuild/dists/jdk1.8" />
+    <property name="checkstyle.version" value="8.28" />
+    <property name="jacoco.version"     value="0.8.5" />
+    <property name="jvmci.home"  location="${lib.dir}/jvmci${home.ext}" />
+    <property name="junit.version" value="4.12" />
     <property name="build.dir"   value="build"/>
     <property name="classes.dir" value="${build.dir}/classes"/>
-    <property name="junit.version" value="4.12" />
 
-    <property name="checkstyle.version" value="8.14" />
+    <property name="mx.cmd" value="../../mx/mx" />
 
     <property environment="env"/>
 
-    <path id="project.classpath">
+<!--    <property name="src.dir"     value="src"/>-->
+<!--    <property name="test.dir"    value="tests"/>-->
+<!--    <property name="lib.dir"     value="libs" />-->
+<!--    <property name="build.dir"   value="build"/>-->
+<!--    <property name="classes.dir" value="${build.dir}/classes"/>-->
+
+
+    <property name="checkstyle.version" value="8.14" />
+
+<!--    <path id="project.classpath">-->
+<!--        <pathelement location="${classes.dir}" />-->
+<!--        <pathelement location="${test.dir}" />-->
+<!--        <pathelement location="${lib.dir}/junit-${junit.version}.jar" />-->
+<!--        <pathelement location="${lib.dir}/hamcrest-core-1.3.jar" />-->
+<!--    </path>-->
+
+    <path id="boot.cp">
+        <pathelement location="${sdk.build}/graal-sdk.jar" />
+        <pathelement location="${truffle.build}/truffle-api.jar" />
+    </path>
+
+    <path id="common.cp">
         <pathelement location="${classes.dir}" />
-        <pathelement location="${test.dir}" />
+        <pathelement location="${svm.build}/svm-core.jar" />
+        <pathelement location="${tools.build}/truffle-profiler.jar" />
+    </path>
+
+<!--    //TODO-change-->
```

Comment:
Change? In which way?

Reply:
The change mentioned was added earlier on but I forgot to remove this comment. Can be removed